MOBILE_API.xml

```diff
@@ -623,6 +623,26 @@
         <element name="SOURCE" since="4.5" />
         <element name="SHUFFLE" since="4.5" />
         <element name="REPEAT" since="4.5" />
+        <!-- Navigation Buttons -->
+        <element name="NAV_CENTER_LOCATION" since="5.2" />
+        <element name="NAV_ZOOM_IN" since="5.2" />
+        <element name="NAV_ZOOM_OUT" since="5.2" />
+        <element name="NAV_PAN_UP" since="5.2" />
+        <element name="NAV_PAN_UP_RIGHT" since="5.2" />
+        <element name="NAV_PAN_RIGHT" since="5.2" />
+        <element name="NAV_PAN_DOWN_RIGHT" since="5.2" />
+        <element name="NAV_PAN_DOWN" since="5.2" />
+        <element name="NAV_PAN_DOWN_LEFT" since="5.2" />
+        <element name="NAV_PAN_LEFT" since="5.2" />
+        <element name="NAV_PAN_UP_LEFT" since="5.2" />
+        <element name="NAV_TILT_TOGGLE" since="5.2" >
+            <description>If supported, this toggles between a top-down view and an angled/3D view. If your app supports different, but substantially similar options, then you may implement those. If you don't implement these or similar options, do not subscribe to this button.</description>
+        </element>
+        <element name="NAV_ROTATE_CLOCKWISE" since="5.2" />
+        <element name="NAV_ROTATE_COUNTERCLOCKWISE" since="5.2" />
+        <element name="NAV_HEADING_TOGGLE" since="5.2" >
```

Comment:
```suggestion
        <element name="NAV_HEADING_TOGGLE" since="5.2" >
```
```suggestion
        <element name="NAV_HEADING_TOGGLE" since="6.0" >
```